django_yadt/management/commands/yadt_refresh_variant.py

```diff
@@ -4,16 +4,21 @@
 
 from ...utils import get_variant
 
+
 class Command(BaseCommand):
-    USAGE = "<app_label> <model> <field> <variant>"
+    def add_arguments(self, parser):
+        parser.add_argument('app_label', type=str)
```

Comment:
Yes, looks like it. Normally I'd prefer that it was more explicit, but I guess it's just noise here so I've taken it out